webpages/js/metaanalyses.js

```diff
@@ -2981,12 +2981,14 @@
   // e.g. ['hidden0', 'col1', 'hidden2', 'hidden3', 'col4'].
   // Passing col1 will unhide hidden0, and passing col4 will unhide 2 and 3.
   function unhideColumns (e) {
-    var index = parseInt(_.findEl(_.findPrecedingEl(e.target, 'th'), '.fullcolinfo.popupbox').dataset.index);
+    var element = _.findEl(_.findPrecedingEl(e.target, 'th'), '.fullcolinfo.popupbox');
+    var index = 0;
 
-    // If we don't have a colIndex, it's the 'add column' button, so start at the end.
-    if (isNaN(index)) {
+    // If element == null or we don't have a colIndex, it's the 'add column' button, so start at the end.
+    if (element == null || ((index = parseInt(element.dataset.index)) && isNaN(index))) {
```

Comment:
if `element` is not null, the index should be there and not be NaN; if it isn't, we'd be hitting a new bug and this code might mask that. In effect, I think this code should just be `if (element == null)`. This may mean the surrounding lines might be further simplified.